legistar/events.py

```diff
@@ -25,6 +21,14 @@ def should_cache_response(self, response):
         return (super().should_cache_response(response) and
                 response.url != self.EVENTSPAGE)
 
+    def key_for_request(self, method, url, **kwargs):
+        # avoid attempting to pull top level events page from cache by
+        # making sure the key for that page is None
```

Comment:
can you add a note that `key_for_request` and `should_cache_response` override scrapelib methods and together have the effect of disabling the cache for the top-level events page? this can be discerned by following the code, but it will save us time in the future to have it noted locally.